packages/flutter_tools/lib/src/device.dart

```diff
@@ -703,9 +704,10 @@ class AndroidDevice extends Device {
         logging.severe('Unexpected response from getprop: "$sdkVersion"');
         return false;
       }
-      if (sdkVersionParsed < 16) {
-        logging.severe('The Android version ($sdkVersion) on the target device '
-            'is too old. Please use a Jelly Bean (version 16 / 4.1.x) device or later.');
+      if (sdkVersionParsed < minApiLevel) {
+        logging.severe(
+          'The Android version ($sdkVersion) on the target device is too old. '
+          'Please use a Jelly Bean (version $minApiLevel / 4.1.x) device or later.');
```

Comment:
"Jelly Bean" and "4.1.x" should probably be in constants next to `sdkVersion` so that when we update one we remember to update the others. (Is API level 16 Jelly Bean and 4.1.x?)